src/js/api/setup-steps.js

```diff
@@ -96,22 +96,26 @@ export function loadSkin(_model) {
 }
 
 export function loadTranslations(_model) {
-    const language = getLanguage();
-    if (language && isTranslationAvailable(language)) {
-        return new Promise((resolve, reject) => {
-            loadJsonTranslation(_model.attributes.base, language, ({ response }) => {
-                // TODO: update localization with translations (JW8-1346)
+    const { attributes } = _model;
+    const { language, base, setupConfig, intl } = attributes;
+    const customLocalization = getCustomLocalization(setupConfig.localization, intl, language);
```

Comment:
`intl` and `language` never change. `setupConfig.localization` is the localization block defined by the developer; `attributes.localization` includes the the developer's customizations added onto the default. Are you saying we should be storing `customLocalization` instead of calling it twice ?

Reply:
OK. Why are you using `setupConfig.localization`. You already have `attributes.localization`.